eng/scripts/Language-Settings.ps1

```diff
@@ -39,7 +39,7 @@ function Get-GoModuleProperties($goModPath)
   $goModPath = $goModPath -replace "\\", "/"
   # We should keep this regex in sync with what is in the azure-sdk repo at https://github.com/Azure/azure-sdk/blob/main/eng/scripts/Query-Azure-Packages.ps1#L227
   # The serviceName named capture group is unused but used in azure-sdk, so it's kept here for parity
-  if ($goModPath -match "(?<modPath>(sdk|profile)/(?<serviceDir>(.*?(?<serviceName>[^/]+)/)?(?<modName>[^/]+$)))")
+  if (!$goModPath.Contains("testdata") -and $goModPath -match "(?<modPath>(sdk|profile)/(?<serviceDir>(.*?(?<serviceName>[^/]+)/)?(?<modName>[^/]+$)))")
```

Comment:
This will need to be updated in azure-sdk as well: https://github.com/Azure/azure-sdk/blob/main/eng/scripts/Query-Azure-Packages.ps1#L238

Reply:
That script discovers modules by iterating the repo's tags. It therefore already filters out `testdata/*` because none of those modules has an associated tag.